tests/Akkling.Tests/app.config

```diff
@@ -53,6 +53,16 @@
     <assemblyIdentity name="Google.Protobuf" publicKeyToken="a7d26565bac4d604" culture="neutral" />
     <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="3.5.1.0" />
   </dependentAssembly>
+  <dependentAssembly>
+    <Paket>True</Paket>
+    <assemblyIdentity name="Microsoft.Extensions.Logging" publicKeyToken="adb9793829ddae60" culture="neutral" />
+    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="1.1.2.0" />
```

Comment:
Hmm. I did not managed to make it work with any version other than 1.1.1.0.

Reply:
I've unlocked tests checking remote deployment scenario between two actor systems communicating over autentic DotNetty transport. It passed.